server/server.go

```diff
@@ -474,6 +475,7 @@ var ConnectorsConfig = map[string]func() ConnectorConfig{
 	"gitlab":          func() ConnectorConfig { return new(gitlab.Config) },
 	"google":          func() ConnectorConfig { return new(google.Config) },
 	"oidc":            func() ConnectorConfig { return new(oidc.Config) },
+	"oauth":           func() ConnectorConfig { return new(oauth.Config) },
```

Comment:
some whitespace issues

Reply:
hmm can't tell where the whitespace is. I took an existing line and just changed the name. It shown no difference.